### ISSUE TITLE
How do I get her to speak Korean?

### ISSUE THREAD

**Issue by joireum** [ardha27/AI-Waifu-Vtuber#92]
hello.

**Comment by joireum** [comment]
it's meaning sound. not a text.

![image](https://github.com/ardha27/AI-Waifu-Vtuber/assets/66953695/7a4973bd-ea15-427c-ab50-aa0eb226feba)

**Comment by ardha27** [comment]
yes, you should try to find TTS that can speak korean and you will need to understand python to apply that new TTS into the code
